ale_linters/elm/make.vim

```diff
@@ -32,10 +37,10 @@ function! ale_linters#elm#make#GetCommand(buffer) abort
         let l:dir_set_cmd = ''
     else
         let l:root_dir = fnamemodify(l:elm_package, ':p:h')
-        let l:dir_set_cmd = 'cd ' . fnameescape(l:root_dir) . '; '
+        let l:dir_set_cmd = 'cd ' . fnameescape(l:root_dir) . ' && '
     endif
 
-    let l:elm_cmd = 'elm-make --report=json --output='.shellescape(g:ale#util#nul_file)
+    let l:elm_cmd = 'elm-make --report=json --output='.shellescape('/dev/null')
```

Comment:
It actually fixes a Windows issue. Elm-make fails for "nul" because it lacks a file type of HTML or js. Instead, it uses "/dev/null" as a flag for not creating an output file.

I'll add a comment denoting as such when I apply the other changes.